README.md

```diff
@@ -356,6 +356,17 @@ config.coverage :CI do |coverage|
 end
 ```
 
+### Ignoring Files
+
+Teaspoon allows defining files and directories to ignore when performing coverage instrumentation.
+
+The below example will ensure that the files within `/test/javascripts` and `/vendor/assets` are not intrumented by Istanbul and will not show up in the coverage results.
+
+```ruby
+config.coverage_ignored_files = ["/test/javascripts", "/vendor/assets"]
```

Comment:
I agree about the option name, I was unsure while I was writing it.
The matcher idea should work here, the dir glob should be able to match the pathname of the assets. I'll have a go and see.